<launch>	
	<env name="ROSCONSOLE_CONFIG_FILE" 
       value="$(find mini_rover)/config/custom_rosconsole.conf"/>
    
    <include file="$(find mini_rover)/launch/dual_cam.launch" />
    <rosparam param="rover_params" command="load" file="$(find mini_rover)/config/rover_config.yaml" />

 	<!-- override the image provide -->
	<param name="rover_params/image_provided" value="False"/>
	<param name="rover_params/show_img" value="False"/>
	
	<node name="rover_cam_persp" pkg="mini_rover" type="img_process_node_pespective.py" output="screen" />
</launch>